internal/app/recordtester/recordtester_app.go

```diff
@@ -341,6 +302,111 @@ func (rt *recordTester) Start(fileName string, testDuration, pauseDuration time.
 	// <-downloader.Done()
 	// glog.Infof(`Pulling stopped after %s`, time.Since(started))
 	// exit(55, fileName, *fileArg, err)
+
+	glog.Info("Record testing done, waiting for Asset to be available")
+	if err = rt.isCancelled(); err != nil {
+		return 0, err
+	}
+
+	assetId := ""
+	wait_interval := 30 * time.Second
+	elapsed_time := 0 * time.Second
+asset:
+	for {
+		glog.Infof("Waiting %s for asset, elapsed=%s", wait_interval, elapsed_time)
+		time.Sleep(wait_interval)
+		elapsed_time = elapsed_time + wait_interval
```

Comment:
Instead of accumulating only the time you sleep, it should be better to save the `starTime := time.Now()` and then log the elapsed time as `time.Since(startTime)`